cli/admin/src/test/java/io/pravega/cli/admin/segmentstore/AbstractSegmentStoreCommandsTest.java

```diff
@@ -77,7 +77,7 @@ public abstract class AbstractSegmentStoreCommandsTest {
     protected static final int CONTAINER_COUNT = 1;
 
     @Rule
-    public final Timeout globalTimeout = new Timeout(60, TimeUnit.SECONDS);
+    public final Timeout globalTimeout = new Timeout(6000, TimeUnit.SECONDS);
```

Comment:
I think this is wrong, please revert this change.

Reply:
@AJadhav29 It is still 6000